feat: add hackernews project in kittehub (#123)
autokitteh/samples: hackernews/README.md

Reviewer: This [README](https://github.com/autokitteh/kittehub/tree/main/aws_health_to_slack) has been approved with the recent requested changes.

Adapt this README to match the format.

---

> title: Hackernews Alert in Slack 

Reviewer 2: ```suggestion
title: Hacker News Alerts in Slack 
```

---

> title: Hackernews Alert with Slack 

Reviewer 2: ```suggestion
title: Hackernews Alert in Slack 
```

> description: A bot that tracks Hacker News articles by topic and sends updates to Slack.

Reviewer 2: ```suggestion
description: Track Hacker News articles by topic and send updates to Slack
```

integrations: ["slack"]
categories: ["Office Automation"]
---

> 

Reviewer 2: Redundant empty line

```suggestion
```

> # Hackernews Slack Sample

Reviewer 2: ```suggestion
# Hacker News Alerts in Slack
```

This project monitors Hacker News for new articles matching a specific topic, fetching their details in real-time. It compares the latest results with previously fetched articles to identify new ones and sends their title and URL as notifications to a Slack channel.

## How It Works

1. Reads the topic from the Slack command.
2. Adds the topic to the search query of HackerNews. Check out https://www.algolia.com/doc/api-reference/rest-api for more details. 

> 3. Returns the new articles related to the topic that are published after the code started.

Reviewer 2: ```suggestion
1. Read the topic from the Slack command
2. Add the topic to the Hacker News search query (check out [Algolia's REST API](https://www.algolia.com/doc/api-reference/rest-api) for more details)
3. Return all the new articles related to the topic that were published since the last check
```

> 

Reviewer 2: Redundant empty line

```suggestion
```

## Deployment & Configuration

### Cloud Usage (Recommended)

> - Initialize your connection with slack through the UI

Reviewer 2: ```suggestion
- Initialize your connection with Slack through the UI
```

> 

Reviewer 2: ```suggestion
```

> ## Trigger Workflow

Reviewer 2: This section belongs *below* prerequisites and install steps

> - Type `/your_slack_app` `topic` in the Slack channel you set in the environment variable, replacing `topic` with what you want to search for, to start tracking articles.

Reviewer 2: ```suggestion
- Type `/your-slack-app topic` in the Slack channel you set in the environment variable, replacing `topic` with what you want to search for, to start tracking articles
```

> - The workflow runs automatically every two minutes after deployment. 

Reviewer 2: ```suggestion
- The workflow runs automatically every two minutes after deployment
```

#### Prerequisites

> - [Install AutoKitteh](https://docs.autokitteh.com/get_started/install)

Reviewer 2: ```suggestion
#### Prerequisites

- [Install AutoKitteh](https://docs.autokitteh.com/get_started/install)
```
